src/polyfill/RTCDataChannel.ts

```diff
@@ -1,57 +1,60 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
 import * as exceptions from './Exception';
 import { DataChannel } from '../lib/index';
+import RTCPeerConnection from './RTCPeerConnection';
+import { RTCErrorEvent } from './Events';
 
 export default class RTCDataChannel extends EventTarget implements globalThis.RTCDataChannel {
     #dataChannel: DataChannel;
     #readyState: RTCDataChannelState;
-    #bufferedAmountLowThreshold: number;
-    #binaryType: BinaryType;
+    #bufferedAmountLowThreshold: number = 0;
+    #binaryType: BinaryType = 'blob';
     #maxPacketLifeTime: number | null;
     #maxRetransmits: number | null;
     #negotiated: boolean;
     #ordered: boolean;
-
-    #closeRequested = false;
+    #pc: RTCPeerConnection;
 
     // events
-    onbufferedamountlow: ((this: RTCDataChannel, ev: Event) => any) | null;
-    onclose: ((this: RTCDataChannel, ev: Event) => any) | null;
-    onclosing: ((this: RTCDataChannel, ev: Event) => any) | null;
-    onerror: ((this: RTCDataChannel, ev: Event) => any) | null;
-    onmessage: ((this: RTCDataChannel, ev: MessageEvent) => any) | null;
-    onopen: ((this: RTCDataChannel, ev: Event) => any) | null;
-
-    constructor(dataChannel: DataChannel, opts: globalThis.RTCDataChannelInit = {}) {
+    onbufferedamountlow: globalThis.RTCDataChannel['onbufferedamountlow'];
+    onclose: globalThis.RTCDataChannel['onclose'];
+    onclosing: globalThis.RTCDataChannel['onclosing'];
+    onerror: globalThis.RTCDataChannel['onerror'];
+    onmessage: globalThis.RTCDataChannel['onmessage'];
+    onopen: globalThis.RTCDataChannel['onopen']
```

Comment:
bettter to let the types define it, rather than manually declare any, a typed event target implementation would be prefered however